docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+services:
+    autopsy:
+        image: bannsec/autopsy:latest
+        container_name: autopsy
+        shm_size: "2gb"
+        command: "/opt/autopsy-4.13.0/bin/autopsy"
```

Comment:
This will probably start failing at some point since autopsy will change their versions. Might need to modify the Dockerfile to standardize the path.

For instance:
- command (and PATH below) would become `/opt/autopsy/bin/autopsy`

https://github.com/bannsec/autopsy_docker/blob/6e9ebd159d6b42946c532f39e35d670bc11389d8/Dockerfile#L10
```
mkdir autopsy && unzip -d autopsy autopsy.zip && mv autopsy/autopsy*/* autopsy/. && rm autopsy.zip && \
```

Might wanna try building with that, but it _should_ work.